README.md

```diff
@@ -160,7 +160,9 @@ Other:
   - [Coda/SubEtha syntax Mode](https://github.com/aaronmccall/jade.mode)
   - [html2pug](https://github.com/donpark/html2jade) converter
   - [pug2php](https://github.com/SE7ENSKY/jade2php) converter
-  - [Pug Server](https://github.com/ded/jade-server)  Ideal for building local prototypes apart from any application
+  - [Jade Server](https://github.com/ded/jade-server) Ideal for building local prototypes apart from any application (Doesn't works with pug files)
+  - [Pug Server](https://github.com/ctrlaltdev/pug-server) Fork of Jade Server that works with pug files
+  - [Pug Compiler](https://github.com/ctrlaltdev/pug-compiler) Pug to HTML 'compiler', ideal to create static websites
```

Comment:
I'm not sure what this provides over our official pug-cli

Reply:
None - I shamefully didn't know about pug-cli.